ticket.go

```diff
@@ -59,13 +59,20 @@ func OpenStore(r io.Reader) (*Store, error) {
 	return s, nil
 }
 
-// Yada.
+// WriteTo writes encodes data to JSON and writes to file.
 func (s *Store) WriteTo(w io.Writer) error {
-	w = s.tickets
-	for _, tk := range w{
-		json.NewEncoder(tk).Encode
+	tks := s.tickets
```

Comment:
We don't actually need a variable `tks` here, do we; we can `range` over `s.tickets` directly.